kentikapi/client.go

```diff
@@ -39,11 +53,20 @@ type Client struct {
 	Plans              *resources.PlansAPI
 	Alerting           *resources.AlertingAPI
 
+	// CloudExportAdminServiceAPI, SyntheticsAdminServiceAPI and SyntheticsDataServiceAPI are http clients
```

Comment:
Since we now have the code on a dev branch, simply remove the HTTP/REST code for all APIs that have been already converted to gRPC. The goal is to convert _all_ existing v6 APIs to gRPC and verify that external behavior is identical to their HTTP/REST counterparts _before_ the `dev_grpc` branch is merged to `master`. Ultimately the change should be entirely transparent to users. They will simply start using gRPC once they rebuild their app with new version of the SDK (Terraform providers will be the guinea pigs :)).

Reply:
Removal tracked in https://clkentik.atlassian.net/browse/KNTK-370